tests/run-make/allow-non-lint-warnings-cmdline/rmake.rs

```diff
@@ -0,0 +1,8 @@
+// Test that -A warnings makes the 'empty trait list for derive' warning go away.
```

Comment:
Discussion: I thought about this a bit, could this be written as a UI test? Something like

```rs
//@ compile-flags: -Awarnings

#[derive()]
#[derive(Copy, Clone)]
pub struct Foo;

pub fn main() {}
```

and then obviously the stderr would be blessed which would not have any warnings?

I'm trying this locally, let me get back to you.

Reply:
Yes it could actually. Do you prefer me to switch this test to a ui test?

Reply:
Yeah, if it doesn't really need the facilities of run-make tests, it's usually better as ui tests.